changes/next/private_sieve_mdn

```diff
@@ -0,0 +1,19 @@
+Description:
+
+The new `private_sieve_mdn` option helps prevent leaking usernames.
+
+Config changes:
```

Comment:
Nit: For consistency should have another new line before "Config changes"

Reply:
In my mind, we have two blank lines before headings in these files, so that single blank lines can be paragraph separators within a section, though in practice the section bodies are often single paragraphs anyway, or even single lines.  It's going to be transcribed to release notes and upgrade docs by a human anyway, so as long as the required information is there (and in this case it is, thanks!), the syntax doesn't matter too much.

I imagine at some point we'll automate collating these changes files into release notes entries, and to support that we'll probably need to be more specific and strict about the contents and syntax of these files, but we're not there yet.

Reply:
I have added the second blank line, but may push back on this in the future, depending on how the future goes!  For now, just lookin' to get it done. :)